blog/04-18-2023-How-to-Store-NFT-Smart-Contract-Assets-Using-The-FileBase/filebase.md

```diff
@@ -0,0 +1,143 @@
+
+![filebase1](https://user-images.githubusercontent.com/105144630/232922583-4f3ebe6e-5ec2-4bc0-a27b-80bb71b28e6f.png)
+
+If your verification was successful, you will be prompted to login and your dashboard will appear.
+
+![filebase2](https://user-images.githubusercontent.com/105144630/232922652-19d92393-534a-4da1-8ea9-a6c4d68a82d2.png)
+
+Go to buckets on the left side of the screen. There are 0 buckets since it is a new account. Click on Create Bucket. 
+
+A modal should pop up asking for the bucket name and the storage network. Since the Smart contract is named POAP, I have decided to use poap-guide as the bucket name and IPFS as the storage network. We should now have a new bucket in the dashboard. 
+
+![Uploading filebase3.png…]()
+
+Click on the bucket name, and begin uploading the files using the upload button provided on the interface. Once uploaded, click on one of the objects/images, and you should be taken to the object overview, where you will see a couple of links, including the IPFS gateway URL, which can be used to access the NFT image. 
+
+![filebase4](https://user-images.githubusercontent.com/105144630/232922848-dc5f7b8b-307a-4af1-8a79-eb14c60c49b0.png)
+
+The following images are the NFT images used for this project. [Rare-NFT1](https://ipfs.filebase.io/ipfs/QmPZmVVcANANaV5mt88iiTPFFpwnN5hcwtEHhT4JKtPZ8s), [Rare-NFT2](https://ipfs.filebase.io/ipfs/QmVjz5WCvaJMMxvtzLgWSCRb8TGaTx4cvMe4cUmGihcBKb), and [Rare-NFT3](https://ipfs.filebase.io/ipfs/QmSy2NUWTJFgFkctMBCBNpnLzVJPWUiViDbBzVkBTJTYVA).
+
+![filebase5](https://user-images.githubusercontent.com/105144630/232922923-828aa717-3791-4341-a1f1-f9dc1e6b670f.png)
+
+We will use these URLs as the URI while minting the POAP NFTs. This is examined in the next chapter.
+
+## Minting the NFT Using the Smart Contract and FileBase
+
+To mint the NFTs, you need to deploy the NFT contract. Once deployed, the contract owner can call the awardAttendee function passing in the receiver's address and the link to one of the NFTs stored on FileBase using the IPFS gateway URL.
+
+The recipient can verify that he owns an NFT on the blockchain and check for the image.
+
+## Conclusion
+FileBase is a web3 Storage-As-A-Service provider which provides a safer and more secure way to store information. It is an aggregation service that gives users access to multiple IPFS services, a feature that makes it the favorite. FiileBase is one of the best options to safeguard your NFT asset for the contract you deployed or from different NFT marketplaces.
+
+## NEXT STEP
+Filebase is not the only decentralized storage solution in Web3. Having gone through this tutorial, you can also check out Arweave and Filecooin. Have fun building.
```

Comment:
Can you add links to help the readers better, finding other related tutorials or videos on celo sage or any other platform